frontend/packages/ceph-storage-plugin/src/plugin.ts

```diff
@@ -22,6 +29,35 @@ const plugin: Plugin<ConsumedExtensions> = [
       flag: CEPH_FLAG,
     },
   },
+  {
+    type: 'Dashboards/Tab',
+    properties: {
+      id: 'persistent-storage',
+      title: 'Persistent Storage',
+    },
+  },
+  {
+    type: 'Dashboards/Card',
+    properties: {
+      tab: 'persistent-storage',
+      position: GridPosition.MAIN,
+      loader: () =>
+        import(
+          './components/dashboard-page/storage-dashboard/health-card' /* webpackChunkName: "storage-health-card" */
```

Comment:
I'd suggest adding plugin ID to `webpackChunkName`, e.g. `ceph-storage-health-card` or similar.